Presentation.Web/app/components/it-contract/it-contract-plan.controller.ts

```diff
@@ -822,7 +824,23 @@
                                 operator: "contains"
                             }
                         }
-                    }
+                    },
+                    {
+                        field: "LastChangedName", title: "Sidst redigeret: Bruger", width: 150,
+                        persistId: "lastchangedname",
+                        template: dataItem => `${dataItem.LastChangedByUser.Name} ${dataItem.LastChangedByUser.LastName}`,
+                        hidden: true,
+                        sortable: false,
+                        filterable: false
```

Comment:
this must be both filterable and sortable.. see how that is done in the it-system-catalog,controller.ts